tests/compute/test_sampling.py

```diff
@@ -272,6 +272,7 @@ def _gen_neighbor_sampling_test_graph(hypersparse, reverse):
         }, {'user': card if card is not None else 4})
         g = g.to(F.ctx())
         g.edata['prob'] = F.tensor([.5, .5, 0., .5, .5, 0., 1.], dtype=F.float32)
+        g.edata['mask'] = F.tensor([True, True, False, True, True, False, True])
```

Comment:
Does this mean that users can simultaneously specify a probability distribution for sampling edges as well as a boolean mask for sampling eligibility? This seems a bit convoluted. What's the motivation for supporting such cases?